### PR TITLE
Add Scheduler::spawn_nowait (ref #13)

### PR DESCRIPTION
**Pull request by adamrothman**
Also adds missing pytest-asyncio dependency. I borrowed the `_nowait` nomenclature from `asyncio.Queue` and friends.

### CODE REVIEW

File: aiojobs/_scheduler.py [hunk 1]
```diff
@@ -108,6 +103,21 @@ def call_exception_handler(self, context):
     def exception_handler(self):
         return self._exception_handler
 
+    def _spawn_nowait(self, coro) -> Job:
+        """Common (synchronous) job-spawning code.
```
Comment (stasfilin) [inline aiojobs/_scheduler.py]: Need enter for doc string :)

File: aiojobs/_scheduler.py [hunk 2]
```diff
@@ -64,21 +64,17 @@ def pending_count(self):
...
-        job = Job(coro, self, self._loop)
-        should_start = (self._limit is None or
-                        self.active_count < self._limit)
-        self._jobs.add(job)
-        if should_start:
-            job._start()
-        else:
-            self._pending.append(job)
-        return job
+        return self.spawn_nowait(coro)
```
Comment (kxepal) [inline aiojobs/_scheduler.py]: What's the difference between `await spawn` and `spawn_nowait`? Both would run syncly without yielding execution context to next coroutine.

File: aiojobs/_scheduler.py [hunk 3]
```diff
@@ -64,21 +64,17 @@ def pending_count(self):
     def closed(self):
         return self._closed
 
-    async def spawn(self, coro):
+    async def spawn(self, coro) -> Job:
```
Comment (asvetlov) [inline aiojobs/_scheduler.py]: We don't use type annotations in the project yet.
Let's make an another issue for it, convert everything and setup `mypy` checks.
Honestly I doubt if the task could be done until `mypy` support for https://www.python.org/dev/peps/pep-0561/

**Comment by asvetlov** [comment]
Outdated after adding a limit for pending jobs count.

**Pull request closed without merging**